databricks_cli/unity_catalog/metastore_cli.py

```diff
@@ -0,0 +1,218 @@
+              help='Unique identifier of the workspace for the metastore assignment.')
+@click.option('--metastore-id', 'metastore_id', required=True, type=MetastoreIdClickType(),
+              help='Unique identifier of the metastore to assign to the workspace.')
+@click.option('--default-catalog-name', 'default_catalog_name', required=False,
+              default='hive_metastore',
+              help='Name of the default catalog to use with the metastore ' +
+                   '(default: "hive_metastore").')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def assign_metastore_cli(api_client, workspace_id, metastore_id, default_catalog_name):
+    """
+    Assign a metastore to a specified workspace.
+
+    Calls the 'createMetastoreAssignment' RPC endpoint of the Unity Catalog service.
+    If that fails due to the workspace already having a Metastore assigned, it calls
+    the 'updateMetastoreAssignment' endpoint.
+    Returns nothing.
+
+    """
+    resp = UnityCatalogApi(api_client).create_metastore_assignment(workspace_id, metastore_id,
+                                                                   default_catalog_name)
+    # resp will just be an empty object ('{}') but it's good to print *something*
+    click.echo(mc_pretty_format(resp))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Unassigns a metastore from a workspace.')
+@click.option('--workspace-id', 'workspace_id', required=True, type=WorkspaceIdClickType(),
+              help='Unique identifier of the workspace.')
+@click.option('--metastore-id', 'metastore_id', required=True, type=MetastoreIdClickType(),
+              help='Unique identifier of the metastore to unassign from the workspace.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def unassign_metastore_cli(api_client, workspace_id, metastore_id):
+    """
+    Unassign a metastore from a workspace.
+
+    Calls the 'deleteMetastoreAssignment' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    resp = UnityCatalogApi(api_client).delete_metastore_assignment(workspace_id, metastore_id)
+    # resp will just be an empty object ('{}') but it's good to print *something*
+    click.echo(mc_pretty_format(resp))
+
+
+def register_metastore_commands(cmd_group):
+    cmd_group.add_command(create_metastore_cli, name='create-metastore')
+    cmd_group.add_command(list_metastores_cli, name='list-metastores')
+    cmd_group.add_command(get_metastore_cli, name='get-metastore')
+    cmd_group.add_command(update_metastore_cli, name='update-metastore')
+    cmd_group.add_command(delete_metastore_cli, name='delete-metastore')
+    cmd_group.add_command(metastore_summary_cli, name='metastore-summary')
+    cmd_group.add_command(assign_metastore_cli, name='assign-metastore')
+    cmd_group.add_command(unassign_metastore_cli, name='unassign-metastore')
```

Comment:
please make command naming consistent with the rest of the CLI (see comments for other command groups)

Reply:
Addressed in #471.